lib/Gocdb_Services/ServiceService.php

```diff
@@ -1993,19 +1993,49 @@ public function deleteEndpoint(\EndpointLocation $endpoint, \User $user) {
         // check user has permission to edit endpoint's service
         $this->validateAddEditDeleteActions ( $user, $service );
 
-        $this->em->getConnection ()->beginTransaction ();
-        try {
-            $serviceDAO = new \ServiceDAO ();
-            $serviceDAO->setEntityManager ( $this->em );
-            $serviceDAO->removeEndpoint ( $endpoint );
+        $this->deleteEndpointLogic($endpoint);
+    }
 
-            $this->em->flush ();
-            $this->em->getConnection ()->commit ();
-        } catch ( \Exception $e ) {
-            $this->em->getConnection ()->rollback ();
-            $this->em->close ();
-            throw $e;
-        }
+    /**
+     * Function called from API to delete an endpoint
+     *
+     * @param  EndpointLocation $endpoint endpoint to be deleted
+     * @param  Service          $service  service of endpoint being deleted
+     * @thows Exception
```

Comment:
The others use 2 spaces.

```suggestion
     * @thows  Exception
```